### PR TITLE
Circle: Add support for Intel ASM syntax

### PR DESCRIPTION
**Pull request by thedmd**
This PR add support for Intel ASM syntax, which was added to 129 build of circle.

### CODE REVIEW

File: etc/config/circle.amazon.properties [hunk 1]
```diff
@@ -1,17 +1,21 @@
...
 defaultCompiler=circlelatest
 demangler=/opt/compiler-explorer/gcc-11.1.0/bin/c++filt
 objdumper=/opt/compiler-explorer/gcc-11.1.0/bin/objdump
 
-#group.circle.intelAsm=-mllvm --x86-asm-syntax=intel
 compiler.circlelatest.exe=/opt/compiler-explorer/circle-latest/circle
 compiler.circlelatest.name=Latest
 compiler.circlelatest.semver=latest
 
+compiler.circle129.exe=/opt/compiler-explorer/circle-129/circle
```
Comment (mattgodbolt) [inline etc/config/circle.amazon.properties:11]: Thanks! In order for circle-129 to be installed, there's a PR needed to our https://github.com/compiler-explorer/infra repo (should be a super simple one to bin/yaml/circle.yaml only). I can do this if you're not comfortable doing so.

File: etc/config/circle.amazon.properties [hunk 2]
```diff
@@ -1,17 +1,21 @@
...
+compiler.circle129.name=Build 129
+compiler.circle129.semver=129
+
 compiler.circle128.exe=/opt/compiler-explorer/circle-128/circle
 compiler.circle128.name=Build 128
 compiler.circle128.semver=128
+compiler.circle128.intelAsm=
```
Comment (mattgodbolt) [inline etc/config/circle.amazon.properties:18]: Nice - thanks for setting this empty to disable intel for the older version :)

**Review by mattgodbolt — APPROVED** [review]
Looks good! Will look to merge after the infra repo's updated (I'll do the change tomorrow if you don't get to it before). Thanks!

File: etc/config/circle.amazon.properties (hunk 1 above)
Comment (mattgodbolt) [reply etc/config/circle.amazon.properties:11]: Oh my! It's already done!! I'm so sorry I missed that! Merging now!

**Comment by mattgodbolt** [comment]
Apologies for the confusion earlier...it's late here...

**Comment by mattgodbolt** [comment]
I'll do a site update tomorrow and update here when it's live! thank you again! :)

**Comment by thedmd** [comment]
Thanks for review&merge. : )

**Comment by mattgodbolt** [comment]
Darn it: I just deployed to our staging instance and it's not working. 
Screenshot from 2021-07-29 07-58-27
...I'll dig in to this a bit later...

**Pull request merged**